codecov.yml

```diff
@@ -8,7 +8,7 @@ coverage:
     project: # settings affecting project coverage
       default:
         target: auto # auto % coverage target
-        threshold: 5%  # allow for 5% reduction of coverage without failing
+        threshold: 10%  # allow for 10% reduction of coverage without failing
```

Comment:
There was so little Python code in the repo up until this point that we were at nearly 100% before.  I think we can bump this threshold back to 5% soon, now that we have a more reasonable amount of code for the coverage tool to check.